modules/ROOT/pages/migration-transports.adoc

```diff
@@ -0,0 +1,42 @@
+= Migrating Transports
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The transports in Mule 3 have been replaced by connectors that follow an operation model in Mule 4. Most transports are
+replaceable by a connector in Mule 4.
+
+This page addresses items of migration that are common to all transports. For transport-specific migration
+documentation, refer to xref:migration-connectors.adoc[Migrating Connectors and Modules to Mule 4] and its child pages.
+
+[[service_overrides]]
+== Service Overrides
+
+In Mule 3, the behavior of a transport could be customized via a `service-overrides` inside the connector, which
+references Java classes with custom code to provide the new functionality.
+
+.Mule 3 Example: Connector with `service-overrides`
+[source,xml,linenums]
+----
+<imap:connector name="imapConnector" moveToFolder="processed" deleteReadMessages="false" checkFrequency="200">
+    <service-overrides messageReceiver="org.mule.transport.email.functional.RetrieveMessageReceiverPollCounter"/>
+</imap:connector>
+----
+
+In Mule 4, this kind of customization through custom Java code is not supported. There are three possible approaches
+that can be taken to migrate such scenario, depending on what the custom code is actually doing:
+
+* Replace it with some other feature of Mule or another component (i.e.: xref:connectors::java/java-module.adoc[Java Module]
+or xref:connectors::scripting/scripting-module.adoc[Scripting Module]) that provides a similar functionality.
+* Create a new connector based on the replacement of the modified transport with the custom code added. This will
+require working with the SDK that was used to implemented the new connector for Mule 4. Refer to xref:1.1@mule-sdk::index.adoc[Mule SDK]
+to get started with the Mule SDKs.
+//* Going through product management to add support for what the custom code does
```

Comment:
This is commented, I would add this to the doc.

Reply:
Yes, i asked @dukesphere for a more appropriate wording for that.

Reply:
👍